components/infobox/commons/infobox_league.lua

```diff
@@ -381,6 +381,10 @@ function League:_definePageVariables(args)
 
 	Variables.varDefine('tournament_type', args.type)
 	Variables.varDefine('tournament_status', args.status or Variables.varDefault('tournament_status'))
+	Variables.varDefine(
```

Comment:
what do we need the var for?

Reply:
That var is set on some wikis in the customs (e.g. sc2 hdb, wc), therefore i thought it would be good to have it always set.

Reply:
afaik it is only set for storing into smw so imo we can kill it since we (hopefully) kill smw this month